assets/model_monitoring/components/src/model_monitor_output_metrics/entities/signals/signal.py

```diff
@@ -46,9 +46,10 @@ def publish_metrics(self, step: int):
             threshold = None
             if "threshold" in run_metric:
                 threshold = run_metric["threshold"]
-            publish_metric(
-                run_metric["runId"], float(run_metric["value"]), threshold, step
-            )
+            if "value" in run_metric and run_metric["value"]:
```

Comment:
just curious in what scenario we'll get a None metric?

Reply:
Actually I don't know, I am asking customer